cocos/2d/CCScene.cpp

```diff
@@ -167,7 +172,7 @@ void Scene::render(Renderer* renderer)
         //clear background with max depth
         camera->clearBackground(1.0);
         //visit the scene
-        visit(renderer, transform, 0);
+        visit(renderer, transform, Node::FLAGS_TRANSFORM_DIRTY);
```

Comment:
HitTest is vritual function in UI::Widget class before, i reused this interface and move it to Node class.  
If Change it to a Helper function like HitTestInNodeContent(...),  i'll move hittest back to UI::Widget.
How about this solution?
- About Node::FLAGS_TRANSFORM_DIRTY problem, i'll try to find out other solution later.